rubble/src/link/ad_structure.rs

```diff
@@ -97,6 +100,10 @@ impl<'a> ToBytes for AdStructure<'a> {
                 buf.write_u8(Type::SHORTENED_LOCAL_NAME)?;
                 buf.write_slice(name.as_bytes())?;
             }
+            AdStructure::ManufacturerSpecificData(data) => {
+                buf.write_u8(Type::MANUFACTURER_SPECIFIC_DATA)?;
+                buf.write_slice(data)?;
```

Comment:
Doesn't this require a company ID before the data?

Reply:
You are right, it would be cleaner to include it in the structure. I'll update it